### PR TITLE
Provide the ability to show guidance view images on CarPlay.

### PR DESCRIPTION
**Pull request by MaximAlien**
### Description

PR provides the ability to show guidance view images on CarPlay. For example:

Screen Shot 2023-01-05 at 10 40 14 PM

### CODE REVIEW

File: Example/ViewController+FreeDrive.swift [hunk 1]
```diff
@@ -153,6 +153,7 @@ extension ViewController {
     }
     
     func edgeNames(identifier: RoadGraph.Edge.Identifier) -> [String] {
+        if navigationMapView == nil { return [] }
```
Comment (MaximAlien) [inline Example/ViewController+FreeDrive.swift:156]: Fixes a rare crash when `navigationMapView` can be unavailable right after starting active navigation.

**Review by ShanMa1991 — APPROVED** [review]
I think we may add the change to the ChangeLog about the ability to support the junction view on CarPlay.

**Comment by MaximAlien** [comment]
> I think we may add the change to the ChangeLog about the ability to support the junction view on CarPlay.

Yep, will add it, thanks. Was also thinking about providing a way to prevent showing junction image (e.g. via some delegate), but will probably leave it out of scope for now.

**Comment by MaximAlien** [comment]
Cherry-picked to `release-v2.10` branch in fd501779f55fd0d134fca16c677671a8cf99ea58.

**Pull request merged**